src/utils/resources/preference/helper.ts

```diff
@@ -0,0 +1,27 @@
+import { V1beta1VirtualMachinePreference } from '@kubevirt-ui/kubevirt-api/kubevirt';
+import { BootMode } from '@kubevirt-utils/components/FirmwareBootloaderModal/utils/constants';
+
+// Addition for 1.4 kubevirt release. We need bump kubevirt version on kbuevirt-api repo
```

Comment:
@avivtur could u take care of it please?

Reply:
we have the pr here: https://github.com/kubevirt-ui/kubevirt-api/pull/61 working on it with aviv

Reply:
https://github.com/kubevirt-ui/kubevirt-api/pull/62 and https://github.com/kubevirt-ui/kubevirt-api/pull/63
should cover this